### ISSUE TITLE
Cloudy data file is corrupt

### ISSUE THREAD

**Issue by trquinn** [grackle-project/grackle_data_files#1]
The `CloudyData_UVB=HM2012_shielded.h5` data file has "0" for the MMW array for the highest density, highest redshift column.
Repeat by (in python with the h5py module):
```
>>> import h5py
>>> f = h5py.File('CloudyData_UVB=HM2012_shielded.h5',mode="r")
>>> cr = f['CoolingRates']
>>> mmw = cr['Primordial']['MMW']
>>> mmw[-1][-1]
array([0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
       0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
       0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
       0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
       0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
       0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
       0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
       0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
       0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
       0., 0., 0., 0., 0., 0., 0., 0.])
```
Those zeros will cause havoc when trying to get a temperature at high density and high redshfit.

**Comment by trquinn** [comment]
The "Cooling" and "Heating" tables also have this problem.

**Comment by brittonsmith** [comment]
Issue resolved in PR #2 and PR [#81](https://github.com/grackle-project/grackle/pull/81).
